src/protocols/ethereum/libs/EthereumAdapter.ts

```diff
@@ -254,4 +256,13 @@ export class EthereumAdapter extends BaseProtocolAdapter {
 
     return { hash: res.transactionHash };
   }
+
+  override getTxAmountTotal(
+    transaction: ITransaction,
+    isReceived: boolean = false,
+  ) {
+    return new BigNumber(transaction.tx?.amount || 0)
+      .plus(isReceived ? 0 : transaction.tx?.fee || 0)
+      .toNumber();
+  }
```

Comment:
I don't think every protocol should have it's own way of calculating the total amount. We need to have one global way of doing this based on the fact that the `ITransaction` has the same values for every protocol. So for example the amount stored within the `ITransaction` is the real amount that don't need any transformations. The transformations should happen in `normalize...` transaction functions.

Same applies to other adapters.